core/src/main/java/kafka/server/share/DelayedShareFetch.java

```diff
@@ -153,13 +133,20 @@ public boolean tryComplete() {
             shareFetchData.groupId(), shareFetchData.memberId(),
             shareFetchData.partitionMaxBytes().keySet());
 
-        topicPartitionDataFromTryComplete = acquirablePartitions();
+        Map<TopicIdPartition, FetchRequest.PartitionData> topicPartitionData = acquirablePartitions();
 
-        if (!topicPartitionDataFromTryComplete.isEmpty())
-            return forceComplete();
-        log.info("Can't acquire records for any partition in the share fetch request for group {}, member {}, " +
-                "topic partitions {}", shareFetchData.groupId(),
-                shareFetchData.memberId(), shareFetchData.partitionMaxBytes().keySet());
+        if (!topicPartitionData.isEmpty()) {
+            replicaManagerFetchDataFromTryComplete = replicaManagerFetchData(topicPartitionData, false);
+            if (!replicaManagerFetchDataFromTryComplete.isEmpty())
+                return forceComplete();
+            log.info("minBytes is not satisfied for the share fetch request for group {}, member {}, " +
```

Comment:
info seems to be too much here, how helpful this log would be in info mode? Can we move it to debug please.